lib/src/main/java/org/exoplatform/social/addons/storage/RDBMSActivityStorageImpl.java

```diff
@@ -866,39 +878,25 @@ public int getNumberOfOlderComments(ExoSocialActivity existingActivity, Long sin
 
   @Override
   public void updateActivity(ExoSocialActivity existingActivity) throws ActivityStorageException {
-    if(existingActivity == null) {
-      throw new IllegalArgumentException("Activity to update cannot be null");
-    }
-
-    ActivityEntity updatedActivity = activityDAO.find(Long.valueOf(existingActivity.getId()));
-
-    if(updatedActivity != null) {
-      if (existingActivity.getId().startsWith(COMMENT_PREFIX)) {
-        // update comment
-        updatedActivity.setUpdatedDate(new Date());
-      } else {
-        //create or remove liker if exist
-        processLikerActivityInStreams(new HashSet<>(Arrays.asList(existingActivity.getLikeIdentityIds())), new HashSet<>(updatedActivity.getLikerIds()), updatedActivity);
-      }
-
-      if (existingActivity.getTitleId() != null) updatedActivity.setTitleId(existingActivity.getTitleId());
-      if (existingActivity.getTitle() != null) updatedActivity.setTitle(existingActivity.getTitle());
-      if (existingActivity.getBody() != null) updatedActivity.setBody(existingActivity.getBody());
-      if (existingActivity.getLikeIdentityIds() != null) updatedActivity.setLikerIds(new HashSet<>(Arrays.asList(existingActivity.getLikeIdentityIds())));
-      if (existingActivity.getMentionedIds() != null) updatedActivity.setMentionerIds(new HashSet<>(Arrays.asList(existingActivity.getMentionedIds())));
-      if (existingActivity.getPermaLink() != null) updatedActivity.setPermaLink(existingActivity.getPermaLink());
-      if (existingActivity.getTemplateParams() != null) updatedActivity.setTemplateParams(existingActivity.getTemplateParams());
-      updatedActivity.setHidden(existingActivity.isHidden());
-      updatedActivity.setComment(existingActivity.isComment());
-      updatedActivity.setLocked(existingActivity.isLocked());
-
-      activityDAO.update(updatedActivity);
+    ExoSocialActivity updatedActivity = getActivity(existingActivity.getId());
```

Comment:
Is this change really related to the PR ? It seems to revert the fix https://github.com/exo-addons/social-rdbms/commit/22b61aeab8e6e8ca9754032728ffd7e5bc63767e